Manager/BackupManager.php

```diff
@@ -71,17 +71,8 @@ public function execute()
             // write log
             $this->logger->critical($e);
 
+            //Should we throw a general exception here? Maybe we should return a bool.
```

Comment:
I think I would prefer to let the execute function return a boolean
It seams irresponsible to Catch everything and just rethrow it... Or is it? 

Is it possible to recover from an exception? If not, we should return bool. If it is, we should throw better exceptions